web/src/cypress/integration/escalationPolicySteps.ts

```diff
@@ -170,7 +171,8 @@ function testSteps(screen: ScreenFormat) {
       cy.get('div[data-cy=slack-chip]').should('contain', '#foobar')
 
       // open edit step dialog
-      cy.get('ul[data-cy=steps-list] :nth-child(1) li')
+      cy.get('ul[data-cy=steps-list] li')
+        .eq(1)
```

Comment:
I think we changed to `:nth-child` to avoid using `eq` due to the retry issues.

Reply:
> .eq() will automatically retry until the element(s) exist in the DOM.

We should be good on that front. `:nth-child` stopped working here, for some reason, but `eq` worked as intended